src/libs/blockchain/gcr/gcr_routines/handleNativeOperations.ts

```
@@ -43,10 +49,127 @@ export class HandleNativeOperations {
                 }
                 edits.push(addEdit)
                 break
-            default:
-                log.warning("Unknown native operation: " + nativePayload.nativeOperation) // TODO Better error handling
-                // throw new Error("Unknown native operation: " + nativePayload.nativeOperation)
+            // REVIEW: TLSNotary attestation request - burns 1 DEM fee, creates token
+            case "tlsn_request":
+                // eslint-disable-next-line no-var
+                var [targetUrl] = nativePayload.args as [string]
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)